src/pages/cart/cart.jsx

```diff
@@ -0,0 +1,153 @@
+import './cart.css';
+// import { useEffect, useState } from "react"
+import { useCart } from '../../context/cart-context';
+import axios from "axios";
+import { useAuth } from '../../context/auth-context';
+
+export function Cart() {
+    const { cartList, wishList, dispatch } = useCart();
+    const { userId, login } = useAuth();
+
+
+    function isItemInWishList(wishList, id) {
+
+        const index = wishList.findIndex((wishlistItem) => wishlistItem._id === id);
+        if (index !== -1) {
+            return true;
+        }
+        return false;
+    }
+    const total = cartList.reduce((accum, item) => {
+        return accum + item.quantity * item.price;
+    }, 0);
+    console.log(cartList)
+
+    async function handleRemove(product) {
+        console.log("handleRemove")
+        try {
+            const response = await axios.delete(`https://neog-ecommerce.herokuapp.com/user/${userId}/product/${product._id}`)
+            if (response.status === 202) {
+                console.log(response)
+                const updatedCart = response.data.user.cart
+                console.log(updatedCart)
+                dispatch({ type: "SET_CART", payload: updatedCart })
+
+
+            }
+
+        } catch (error) {
+            console.log(error)
+        }
+
+    }
+    async function handleMoveToWishlist(product) {
+        if (isItemInWishList(wishList, product._id) === false) {
+            console.log("Item was not in wishlist")
+            console.log(userId)
+            const response = await axios.post(`https://neog-ecommerce.herokuapp.com/user/${userId}/wishlist/${product._id}`);
+            console.log(response.data);
+            console.log(product)
+            dispatch({
+                type: "ADD_TO_WISHLIST",
+                payload: product
+            })
+        }
+
+        console.log("Item is in wishlist")
+        const response = await axios.delete(`https://neog-ecommerce.herokuapp.com/user/${userId}/product/${product._id}`)
+        if (response.status === 202) {
+            console.log(response)
+            const updatedCart = response.data.user.cart
+            console.log(updatedCart)
+            dispatch({ type: "SET_CART", payload: updatedCart })
+
+        }
```

Comment:
Hi Shahazad,
U already have a `handleRemove` function . You can just call it here `handleremove( )`